salt/etc-hosts/hosts.jinja

```diff
@@ -3,12 +3,13 @@
 ### service names ###
 # set the apiserver for 127.0.0.1 on all hosts as haproxy is listening on all nodes
 # and forwarding connections to the real master
-{% if "kube-master" in salt['grains.get']('roles', []) %}
-127.0.0.1 api api.{{ pillar['internal_infra_domain'] }}{% if not salt['caasp_filters.is_ip'](external_fqdn) %} {{ external_fqdn }}{% endif %}
+{% set internal_fqdn = "api api." + pillar['internal_infra_domain'] %}
```

Comment:
Minor comment: maybe it would be better to define both `internal_fqdn` and `external_fqdn` together at the start of this file...

Reply:
Ok, good idea, thank you